Wk2 172 add webhooks guide (#13)
wealthkernel/api-documentation-guides: docs/webhooks/FAQs.md

# Webhooks FAQs

## Do you guarantee ordered events?

We do not guarantee in order events, so you’ll need to consider how to deal with receiving events that are out of order. 

## How many times will you deliver a webhook?

> In order to guarantee that each webhook request is delivered at least once while remaining performant, it is possible that a webhook request may be delivered more than once. 

Reviewer: ```suggestion
We guarantee at least once delivery within our retry policy, so it is possible that a webhook request may be delivered more than once.
```
